web/cobrands/fixmystreet/fixmystreet.js

```diff
@@ -12,6 +12,22 @@ function isR2L() {
     return !!$('html[dir=rtl]').length;
 }
 
+function make_multi(id) {
+    var $id = $('#' + id),
+        all = $id.data('all');
+        presets = $id.data('presets') === undefined ? [{
```

Comment:
Minor nitpick (sorry): this should be unindented to match line 16, or line 17 should end with a comma not a semicolon. (Though I don't know if the ternary expression would do the right thing in that case).